src/devices_models/devices/common/range_constraint.jl

```diff
@@ -194,13 +366,77 @@ function add_reserve_range_constraints!(
     model::DeviceModel{V, W},
     X::Type{<:PM.AbstractPowerModel},
     feedforward::Union{Nothing, AbstractAffectFeedForward},
+) where {V <: PSY.Component, W <: AbstractDeviceFormulation}
+    # TODO: remove this function? use the impl for dispatch.
```

Comment:
?

Reply:
There’s two methods of the functions where the body is the exact same.

Reply:
I made the comment so that I could look into it later.

Reply:
ok